packages/amplify-e2e-tests/src/__tests__/function_11.test.ts

```diff
@@ -0,0 +1,150 @@
+import {
```

Comment:
can you please format this file as it has a lot of end line spaces and incorrect indentation + more than one new line breaks